modules/shared_data/src/main/scala/org/elpaca_metagraph/shared_data/Utils.scala

```diff
@@ -23,6 +24,12 @@ object Utils {
     (balance * 10e7).toLong
   }
 
+  def toTokenAmountFormat(
+    amount: Amount
+  ): Long = {
+    (amount.value.value * 10e7).toLong
+  }
+
```

Comment:
DRY - don't repeat yourself. you've specified already have to map a `Long` to token amount format. use it here